runtime/vm/vmprops.c

```diff
@@ -572,39 +573,35 @@ initializeSystemProperties(J9JavaVM * vm)
 		}
 	}
 
-#if JAVA_SPEC_VERSION < 12
-	/* Following system properties are defined via java.lang.VersionProps.init(systemProperties) and following settings within System.ensureProperties() */
-	{
+	if (JAVA_SPEC_VERSION == 8) {
+		specificationVersion = "1.8";
+	} else {
+#define J9_STR_(x) #x
+#define J9_STR(x) J9_STR_(x)
```

Comment:
We don't need two levels of this macro here as we're only working with a constant.
```c
#define J9_STR(x) #x 
```

Reply:
It appears it still requires two levels of macro [1] otherwise the system property value would be `JAVA_SPEC_VERSION`. 

[1] https://gcc.gnu.org/onlinedocs/gcc-4.8.5/cpp/Stringification.html

Reply:
You're right.  I had misread the doc